browser/mapping.go

```diff
@@ -678,8 +683,8 @@ func mapBrowserType(vu moduleVU, bt api.BrowserType) mapping {
 	}
 }
 
-func panicIfInternalError(ctx context.Context, err error) {
-	if errors.Is(err, k6error.ErrInternal) {
-		k6ext.Panic(ctx, err.Error())
+func panicIfFatalError(ctx context.Context, err error) {
+	if errors.Is(err, k6error.ErrFatal) {
+		go k6ext.Panic(ctx, err.Error())
```

Comment:
Could you clarify why do we need to run this in a separate goroutine? I didn't quite understand it from the commit message.
In any case, should we add a comment here explaining the reason?

Reply:
Good point. Added a fixup with a comment in 350dd34542d57341a2d2754db92270e5efbbfb0e.

Reply:
This isn't relevant since we're now working with [abort](https://github.com/grafana/k6/blob/0b1fe61c5762c83c52a2378f6cdf9c0c3ff1f53d/js/modules/k6/execution/execution.go#L168-L175).